### PR TITLE
fix(stdlib): Fixed memory leak in hash

### PR DESCRIPTION
**Pull request by cician**

### CODE REVIEW

File: stdlib/hash.gr [hunk 1]
```diff
@@ -227,13 +227,17 @@ let rec hashOne = (val, depth) => {
  *
  * @since v0.1.0
  */
-export let hash = (anything) => {
+export let rec hash = (anything) => {
```
Comment (ospencer) [inline stdlib/hash.gr:230]: Looks like we should also be decRefing `anything`, right?

Comment (cician) [reply stdlib/hash.gr:230]: Right.

**Review by ospencer — APPROVED** [review]
Perfect!

**Review by phated — APPROVED** [review]
🔥

**Pull request merged**